src/assembly/all-dependencies.xml

```diff
@@ -14,22 +14,9 @@
     </containerDescriptorHandlers>
     <dependencySets>
         <dependencySet>
-            <outputDirectory>/</outputDirectory>
-            <useProjectArtifact>false</useProjectArtifact>
             <unpack>true</unpack>
-            <unpackOptions>
-                <!-- We exclude this file here to change the order od registered dialects' factories: first should be RLS, then Exasol.-->
-                <excludes>
-                    <exclude>com.exasol.adapter.dialects.SqlDialectFactory</exclude>
-                </excludes>
-            </unpackOptions>
             <scope>runtime</scope>
+            <outputDirectory>/</outputDirectory>
         </dependencySet>
     </dependencySets>
-    <fileSets>
-        <fileSet>
-            <directory>${project.build.outputDirectory}</directory>
-            <outputDirectory>/</outputDirectory>
-        </fileSet>
```

Comment:
Not sure if we need to restore this and add an exception to project-keeper?

Reply:
did so